memsql-plugin/pom.xml

```diff
@@ -102,6 +102,10 @@
           </instructions>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>io.cdap</groupId>
+        <artifactId>cdap-maven-plugin</artifactId>
+      </plugin>
     </plugins>
   </build>
 </project>
```

Comment:
an existing issue, but seems like this file needs a newline at the end.

Reply:
done :)